docs/erd.md

```diff
@@ -0,0 +1,5 @@
+![TM ERD](https://user-images.githubusercontent.com/12103383/56465906-1aa3fb80-6426-11e9-90e4-bf13e5b1e1ce.png)
+
+_[Collaborate over draw.io](https://www.draw.io/#G17julelLWpYwOptKBFhBJ-Uz4PJdKubzL)_
+
+_[Source file](assets/TM ERD.io)_
```

Comment:
Please correct the filename. And it is [not getting rendered on Github as a link](https://github.com/hotosm/tasking-manager/blob/9daf31631a9d161b51b4e89ce4a522c80821c4bd/docs/erd.md). I would try to omit the white space. Maybe that helps.